posthog/models/kafka_engine_dlq/sql.py

```diff
@@ -0,0 +1,21 @@
+KAFKA_ENGINE_DLQ_BASE_SQL = """
+CREATE MATERIALIZED VIEW {database}.{kafka_table_name}_dlq
+(
+    `topic` String,
+    `partition` Int64,
+    `offset` Int64,
+    `raw` String,
+    `error` String
+)
+ENGINE = MergeTree
```

Comment:
So every replica on every shard will have its own copy of the dlq, with no replication?

That doesn't seem ideal.

Reply:
Ah yup sorry some copy pasta from my testing - will make this replicated